src/renderer/index.html

```diff
@@ -19,7 +19,7 @@ <h2 id="title"></h2>
       <div id="loader"></div>
       <p id="status"></p>
       <div id="open"></div>
-      <pre id="cli-output"></pre>
+      <div id="cli-output"></div>
```

Comment:
Can we keep this as a `pre`? I'm guessing the reason this changed is due to test failures -- presumably the line wrapping that Composer might do (if any) can break the test, which is looking for a continuous, unbroken string. Am I right?